STL_Extension/include/CGAL/span.h

```diff
@@ -0,0 +1,405 @@
+// Copyright 2019-2023 Glen Joseph Fernandes
+// (glenjofe@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (http://www.boost.org/LICENSE_1_0.txt)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: BSL-1.0
+//
+// NOTE: This file was taken from boost 1.83 for use in the Orthtree package.
+
+#ifndef BOOST_CORE_SPAN_HPP
+#define BOOST_CORE_SPAN_HPP
+
+#include <CGAL/STL_Extension/internal/boost/data.h>
+
+#include <array>
+#include <iterator>
+#include <type_traits>
+
+namespace boost {
+
+constexpr std::size_t dynamic_extent = static_cast<std::size_t>(-1);
+
+template<class T, std::size_t E = dynamic_extent>
+class span;
```

Comment:
Why do we need this fork?

Reply:
std::span is c++20

Reply:
I cannot see where this is used in the Orthtree.

Reply:
range of points set AFAIR

Reply:
Node_data in Orthtree_traits_point.h is `boost::iterator_range<typename PointRange::iterator>`.
span.h was included in Orthtree, but can be removed and everything still compiles.

The word span is used a few times in the doc in OrthtreeTraitsWithData. Is there any difference between a span and a random access range?